terraform/deployments/govuk-publishing-infrastructure/csvs_s3.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_iam_policy" "csvs_bucket_polices" {
+  name        = "csvs_s3"
+  description = "Read and write to this environment's CSVS s3 buckets"
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Effect = "Allow"
+        Action = [
+          "s3:GetBucketLocation",
+          "s3:ListBucket",
+        ]
+        Resource = [
+          "arn:aws:s3:::${var.govuk_environment}content-data-csvs",
+          "arn:aws:s3:::${var.govuk_environment}-specialist-publisher-csvs",
+          "arn:aws:s3:::${var.govuk_environment}-support-api-csvs",
+          "arn:aws:s3:::${var.govuk_environment}-whitehall-csvs"
+        ]
+      },
+      {
+        Effect = "Allow"
+        Action = [
+          "s3:*MultipartUpload*",
+          "s3:*Object",
+          "s3:*ObjectAcl",
+          "s3:*ObjectVersion",
+          "s3:GetObject*Attributes",
+        ]
+        Resource = [
+          "arn:aws:s3:::${var.govuk_environment}content-data-csvs",
+          "arn:aws:s3:::${var.govuk_environment}-specialist-publisher-csvs",
+          "arn:aws:s3:::${var.govuk_environment}-support-api-csvs",
+          "arn:aws:s3:::${var.govuk_environment}-whitehall-csvs"
```

Comment:
I think these need the `/*` suffixes so that they match objects in the buckets (as opposed to just the buckets themselves). IAM is super annoying like that unfortunately.

Reply:
Damn, your right i copy pasted over it when i changed the format.